From 28406ec9430f436b19df68383a2d46fd998087b2 Mon Sep 17 00:00:00 2001
From: Jeff Lin <jeffulin@google.com>
Date: Fri, 27 Sep 2024 15:30:35 +0800
Subject: [PATCH] Revert "Fix 32bit ARM Linux build for Cast."

The fix is for CastOS only.

This reverts commit afd8319889fd6e5873a43c07889490e356456732.

Change-Id: If858cef9ec6c68751d7af2d1cdef076f817db71e
---
 base/time/time_exploded_posix.cc | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/base/time/time_exploded_posix.cc b/base/time/time_exploded_posix.cc
index 6f4ead4fb2..8f8cf67daf 100644
--- a/base/time/time_exploded_posix.cc
+++ b/base/time/time_exploded_posix.cc
@@ -142,8 +142,7 @@ void Time::Explode(bool is_local, Exploded* exploded) const {
   // Split the |millis_since_unix_epoch| into separate seconds and millisecond
   // components because the platform calendar-explode operates at one-second
   // granularity.
-  auto seconds = base::checked_cast<SysTime>(millis_since_unix_epoch /
-                                             Time::kMillisecondsPerSecond);
+  SysTime seconds = millis_since_unix_epoch / Time::kMillisecondsPerSecond;
   int64_t millisecond = millis_since_unix_epoch % Time::kMillisecondsPerSecond;
   if (millisecond < 0) {
     // Make the the |millisecond| component positive, within the range [0,999],
-- 
2.46.1.824.gd892dcdcdd-goog

